Documentation/Contributing/development-flow.md

```diff
@@ -77,6 +77,11 @@ git remote -v
 
 Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
 
+### Auto assign Issue
+
+You can auto-assign any issue that is been created in Rook Upstream repo, by commenting
+`\assign` in as a issue comment.
```

Comment:
Does it just need to contain, or does it need to be the only text in the comment?
```suggestion
an issue comment with `/assign` in the body.
```